ui/app/AppLayouts/Chat/ChatColumn/MessageComponents/TransactionBubble.qml

```diff
@@ -0,0 +1,128 @@
+import QtQuick 2.3
+import "../../../../../shared"
+import "../../../../../imports"
+import "./TransactionComponents"
+
+Rectangle {
+    property string tokenAmount: "100"
+    property string token: "SNT"
+    property string fiatValue: "10 USD"
```

Comment:
Does the fiat value has the currency symbol as part of the numeric value? or should this be a separate variable?

Reply:
I'm not sure yet what it will look like when it's hooked. So for now, I just created the prop as the display value needs. So as simple as possible